src/js/react/apps/school-search/components/ResultCard.tsx

```diff
@@ -26,10 +27,39 @@ const ResultCard = ({
   let cardImage;
 
   if (imageOverride) {
-   cardImage = <CardImage
+    const sources = [
```

Comment:
Combine the sources variables in to a one template

Reply:
I combined all of them :) So this is fixed 🦖